src/main/kotlin/com/cognifide/gradle/aem/common/build/ProgressIndicator.kt

```diff
@@ -38,7 +38,7 @@ class ProgressIndicator(private val project: Project) {
     private fun <T> ProgressIndicator.launchAsync(block: ProgressIndicator.() -> T): T {
         return runBlocking {
             val blockJob = async(Dispatchers.Default) { block() }
-            val loggerJob = async(Dispatchers.Default) {
+            val loggerJob = async(Dispatchers.IO) {
```

Comment:
fix for nested progressIndicators ;) 

default dispatcher is using max n threads which equals number of CPU cores.
indicator by itself is creating 2 threads ; when nested we need multiple it by 2, then if  running package deploy in parallel we exceeded no of threads reserved for default kotlin coroutine dispatcher.

when we change it to IO dispatcher it takes much bigger no of threads (in docs there is mentioned 64) so that progress indicator never hangs on logging. previosly it was hanging... ;) we saved few threads and avoided deadlock because of exhausted threads on dispatcher